packages/lit/src/element/block-element.ts

```diff
@@ -67,6 +67,27 @@ export class BlockElement<
   @property({ attribute: false })
   page!: Page;
 
+  @property({ attribute: false })
+  path!: string[];
+
+  get pathname(): string {
+    return this.path.join('|');
+  }
+
+  get parentPath(): string[] {
+    return this.path.slice(0, -1);
+  }
+
+  parentBlockElement = <Block extends BlockElement>(): Block | undefined => {
```

Comment:
This looks like a getter? Also the `T | null` typing should be more favorable in our codebase.

Reply:
Updated